client/src/store/list-data/request-action-mapping.ts

```diff
@@ -15,7 +15,8 @@ export const bcMapping: RequestActionMappingI = {
 export const xproMapping: RequestActionMappingI = {
   ASSUMED: ['XCR', 'RLC', 'XUL'],
   REN: ['XCR', 'XCP', 'RLC', 'XUL'],
-  REH: ['XCR', 'XCP', 'RLC', 'XUL']
+  REH: ['XCR', 'XCP', 'RLC', 'XUL'],
+  AML: ['XCR', 'XCP', 'XUL', 'XLP', 'XLL', 'XSO']
```

Comment:
Sorry. You will also need to update the entity selection screen for CANADA and FOREIGN for these entity types and only allow only for XCR, XCP, XUL and AML request_action.